KsApi/extensions/MockGraphQLClient.swift

```diff
@@ -73,6 +88,18 @@ private func producer<T, E>(for property: Result<T, E>?) -> SignalProducer<T, E>
   }
 }
 
+private func producer<T, E>(for property: Result<T, E>?) -> AnyPublisher<T, E> {
```

Comment:
This is just copying the old pattern for ReactiveSwift in `MockGraphQLClient`. Keeping it as close as possible.